src/contracts/misc/GhoCcipSteward.sol

```diff
@@ -0,0 +1,152 @@
+import {RiskCouncilControlled} from './RiskCouncilControlled.sol';
+import {UpgradeableLockReleaseTokenPool, RateLimiter} from './deps/Dependencies.sol';
+
+/**
+ * @title GhoCcipSteward
+ * @author Aave Labs
+ * @notice Helper contract for managing parameters of the CCIP token pools
+ * @dev Only the Risk Council is able to action contract's functions, based on specific conditions that have been agreed upon with the community.
+ */
+contract GhoCcipSteward is Ownable, IGhoCcipSteward, RiskCouncilControlled {
+  using EnumerableSet for EnumerableSet.AddressSet;
+
+  /// @inheritdoc IGhoCcipSteward
+  uint256 public constant MINIMUM_DELAY = 2 days;
+
+  /// @inheritdoc IGhoCcipSteward
+  address public immutable GHO_TOKEN;
+
+  /// @inheritdoc IGhoCcipSteward
+  address public immutable GHO_TOKEN_POOL;
+
+  /// @inheritdoc IGhoCcipSteward
+  address public immutable RISK_COUNCIL;
+
+  mapping(address => uint40) _facilitatorsBucketCapacityTimelocks;
+
+  mapping(address => bool) internal _controlledFacilitatorsByAddress;
+  EnumerableSet.AddressSet internal _controlledFacilitators;
+
+  /**
+   * @dev Only methods that are not timelocked can be called if marked by this modifier.
+   */
+  modifier notTimelocked(uint40 timelock) {
+    require(block.timestamp - timelock > MINIMUM_DELAY, 'DEBOUNCE_NOT_RESPECTED');
+    _;
+  }
+
+  /**
+   * @dev Constructor
+   * @param owner The address of the owner of the contract
+   * @param ghoTokenPool The address of the Gho CCIP Token Pool
+   * @param riskCouncil The address of the risk council
+   */
+  constructor(
+    address owner,
+    address ghoToken,
+    address ghoTokenPool,
+    address riskCouncil
+  ) RiskCouncilControlled(riskCouncil) {
+    require(owner != address(0), 'INVALID_OWNER');
+    require(ghoToken != address(0), 'INVALID_GHO_TOKEN');
+    require(ghoTokenPool != address(0), 'INVALID_GHO_TOKEN_POOL');
+
+    RISK_COUNCIL = riskCouncil;
+    GHO_TOKEN = ghoToken;
+    GHO_TOKEN_POOL = ghoTokenPool;
+
+    _transferOwnership(owner);
+  }
+
+  /// @inheritdoc IGhoCcipSteward
+  function updateFacilitatorBucketCapacity(
+    address facilitator,
+    uint128 newBucketCapacity
+  ) external onlyRiskCouncil notTimelocked(_facilitatorsBucketCapacityTimelocks[facilitator]) {
+    require(_controlledFacilitatorsByAddress[facilitator], 'FACILITATOR_NOT_CONTROLLED');
+    (uint256 currentBucketCapacity, ) = IGhoToken(GHO_TOKEN).getFacilitatorBucket(facilitator);
+    require(
+      _isIncreaseLowerThanMax(currentBucketCapacity, newBucketCapacity, currentBucketCapacity),
+      'INVALID_BUCKET_CAPACITY_UPDATE'
+    );
+
+    _facilitatorsBucketCapacityTimelocks[facilitator] = uint40(block.timestamp);
+
+    IGhoToken(GHO_TOKEN).setFacilitatorBucketCapacity(facilitator, newBucketCapacity);
+  }
+
+  /// @inheritdoc IGhoCcipSteward
+  function updateBridgeLimit(uint256 newBridgeLimit) external onlyRiskCouncil {
+    UpgradeableLockReleaseTokenPool(GHO_TOKEN_POOL).setBridgeLimit(newBridgeLimit);
```

Comment:
This will break if using remote token pools. We can prevent this by using a flag `bridgeLimitEnabled` or something

Reply:
What is the desired behavior if someone tries to call this on a remote pool? It will currently just revert because remote pools do not have the setBridgeLimit function. 

Do we want to add a flag to allow the transaction to succeed but have no effect?

Reply:
I think better to have a flag and informative revert msg.

Reply:
I can do it, noting to myself that since we won't grant the corresponding role to the Steward deployed on the network, I must ensure it fails before calling the function in order to properly show that error from the flag instead of the access control

Reply:
Done